src/couch_doc.erl

```diff
@@ -125,7 +125,14 @@ doc_to_json_obj(#doc{id=Id,deleted=Del,body=Body,revs={Start, RevIds},
     }.
 
 from_json_obj({Props}) ->
```

Comment:
Is this the right place to be making this change? `from_json_obj` is used in places other than when parsing HTTP request bodies. For example, this change will cause `fabric:cleanup_index_files` to fail if any existing design documents are larger than the limit.